python/python/glide/async_commands/core.py

```diff
@@ -2699,6 +2699,70 @@ async def geosearch(
             await self._execute_command(RequestType.GeoSearch, args),
         )
 
+    async def geosearchstore(
+        self,
+        destination: str,
+        source: str,
+        search_from: Union[str, GeospatialData],
+        search_by: Union[GeoSearchByRadius, GeoSearchByBox],
+        count: Optional[GeoSearchCount] = None,
+        store_dist: bool = False,
+    ) -> int:
+        """
+        Stores the results of a geospatial search into a destination sorted set.
```

Comment:
we could be a bit more descriptive with the operation.  
Example: zunionsstore:
https://github.com/aws/glide-for-redis/pull/1550/files#diff-37602a66f00251738190a1e0ebddd5d027022433028ab20ba728852a05abab7eR1978-R1980

Maybe: 
```
        Searches for members in a sorted set stored at `key` representing geospatial data within a circular or rectangular area and stores the result in `destination`.  If `destination` already exists, it is overwritten. Otherwise, a new sorted set will be created.

To get the result directly, see `geosearch`.
```

Reply:
I agree, thanks!